### PR TITLE
Namespace dnd

### PR DESCRIPTION
**Pull request by amolenaar**
DnD from/within the namespace view works.

Fixed scenario:

 * Create a package below the main package, move the diagram in that package.
 * Select the toplevel package and drop it on the diagram canvas
 * Select an element from the toolbox and drop it on the diagram

### PR Checklist
Please check if your PR fulfills the following requirements:

- [x] I have read, and am following the [Contributer guide](https://github.com/gaphor/gaphor/blob/master/CONTRIBUTING.md)
- [x] I have read and understand the [Code of Conduct](https://github.com/gaphor/gaphor/blob/master/CODE_OF_CONDUCT.md)

### PR Type
What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [x] Refactoring (no functional changes, no api changes)
- [ ] Documentation content changes

### CODE REVIEW

File: gaphor/ui/namespace.py [hunk 1]
```diff
@@ -504,7 +504,7 @@ def search_func(model, column, key, iter, data=None):
             Gtk.DestDefaults.ALL, [NamespaceView.DND_TARGETS[-1]], Gdk.DragAction.MOVE
         )
         self.connect("drag-motion", NamespaceView.on_drag_motion)
-        self.connect("drag-drop", NamespaceView.on_drag_drop)
+        # self.connect("drag-drop", NamespaceView.on_drag_drop)
```
Comment (danyeaw) [inline gaphor/ui/namespace.py]: Can we remove this line since you deleted the method?

Comment (amolenaar) [reply gaphor/ui/namespace.py]: > These changes look really good. The drag and drop is now working great in the Namespace. I did notice now when dragging from the toolbox that it is dropping two of each item on to the canvas.

I can not reproduce this behaviour on my side (GTK+3 on MacOS).

**Comment by amolenaar** [comment]
> These changes look really good. The drag and drop is now working great in the Namespace. I did notice now when dragging from the toolbox that it is dropping two of each item on to the canvas.

I can not reproduce this behaviour on my side (GTK+3 on MacOS).

**Comment by danyeaw** [comment]
two-classes

**Comment by amolenaar** [comment]
You first click the class icon, and then DnD it on the canvas?

**Comment by danyeaw** [comment]
toolbox-copier

I think that was just an artifact of me clicking to get focus using the GIF recorder. Maybe this is a better demo. I also show that clicking on the icon and then clicking on the canvas doesn't do anything. To get the duplicates I am just clicking and dragging and releasing on the canvas.

**Comment by amolenaar** [comment]
I have t try on Linux, I think.

**Comment by amolenaar** [comment]
BTW. What do you use to make those GIF recordings?

**Comment by amolenaar** [comment]
BTW1. I  think I nailed the issue :)

**Comment by danyeaw** [comment]
@amolenaar I created the GIFs using [Peek](https://github.com/phw/peek). I think the equivalent for macOS is maybe [LICEcap](https://www.cockos.com/licecap/) or https://gifox.io if you are willing to pay :dollar: .

I pulled down your latest changes and I am getting a MRO TypeError.

```
Traceback (most recent call last):
  File "setup.py", line 138, in <module>
    build_mo=dict(all_linguas=",".join(LINGUAS)),
  File "/home/dan/.cache/pypoetry/virtualenvs/gaphor-py3.7/lib/python3.7/site-packages/setuptools/__init__.py", line 129, in setup
    return distutils.core.setup(**attrs)
  File "/home/dan/.pyenv/versions/3.7.1/lib/python3.7/distutils/core.py", line 148, in setup
    dist.run_commands()
  File "/home/dan/.pyenv/versions/3.7.1/lib/python3.7/distutils/dist.py", line 966, in run_commands
    self.run_command(cmd)
  File "/home/dan/.pyenv/versions/3.7.1/lib/python3.7/distutils/dist.py", line 985, in run_command
    cmd_obj.run()
  File "./utils/command/run.py", line 147, in run
    starter()
  File "./gaphor/__init__.py", line 103, in main
    launch(model)
  File "./gaphor/__init__.py", line 33, in launch
    Application.init()
  File "./gaphor/application.py", line 55, in init
    self.load_services(services)
  File "./gaphor/application.py", line 79, in load_services
    cls = ep.load()
  File "/home/dan/.cache/pypoetry/virtualenvs/gaphor-py3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2324, in load
    return self.resolve()
  File "/home/dan/.cache/pypoetry/virtualenvs/gaphor-py3.7/lib/python3.7/site-packages/pkg_resources/__init__.py", line 2330, in resolve
    module = __import__(self.module_name, fromlist=['__name__'], level=0)
  File "./gaphor/services/copyservice.py", line 12, in <module>
    from gaphor.UML import Element
  File "./gaphor/UML/__init__.py", line 1, in <module>
    from gaphor.UML import modelfactory as model
  File "./gaphor/UML/modelfactory.py", line 12, in <module>
    from gaphor.UML.uml2 import *
  File "./gaphor/UML/uml2.py", line 88, in <module>
    class Parameter(TypedElement, MultiplicityElement, ConnectableElement): pass
TypeError: Cannot create a consistent method resolution
order (MRO) for bases TypedElement, MultiplicityElement, ConnectableElement
```

**Comment by danyeaw** [comment]
@amolenaar I looked at this a little more this evening. It looks 4dce4cf3 is what introduced the TypeError when you saved the uml2.gaphor. The previous version of the generated uml2.py had `class Parameter(TypedElement, MultiplicityElement): pass`, and the new version subclasses the ConnectableElement as well, which is causing the MRO issues.

**Comment by amolenaar** [comment]
@danyeaw This is okay to merge, right?

**Comment by danyeaw** [comment]
@amolenaar I am still not able to launch this version, I am getting the `TypeError: Cannot create consistent MRO` that I mentioned above.

**Comment by amolenaar** [comment]
@danyeaw I removed the change to the `uml2.gaphor` file.

**Pull request merged**